utils/vmCodeExecutor.test.js

```diff
@@ -0,0 +1,30 @@
+import executeVmCode from "./vmCodeExecutor";
+
+jest.mock("./vmCodeExecutor");
+
+describe("executeVmCode", () => {
+  it("유저 함수 코드가 정상적으로 실행되어야 한다.", async () => {
+    const fixture = `function userCode() {
+      return 5 + 2;
+    }
+    userCode();`;
```

Comment:
```suggestion
const fixture = `
  function userCode() {
    return 5 + 2;
  }
  userCode();
`;
```

멀티라인인 문자열의 가독성을 높이기 위해 템플릿리터럴을 적용하신 것 같은데요!

가독성의 목적으로 사용하셨다면  추가적으로 위와 같은 방식으로 개선해보는건 어떨까요? 🤔

Reply:
알려주신 방식으로 작성 해보니 가독성이 훨씬 좋아졌습니다👍

수정사항 반영하겠습니다.